src/DripsHub.sol

```diff
@@ -65,6 +65,21 @@ abstract contract DripsHub {
     /// @notice Number of bits in the sub-account part of userId
     uint256 public constant BITS_SUB_ACCOUNT = 224;
 
+    /// @notice Emitted when an account is created
+    /// @param accountId The account ID
+    /// @param owner The account owner
+    event AccountCreated(uint32 indexed accountId, address indexed owner);
+
+    /// @notice Emitted when an account ownership is transferred.
+    /// @param accountId The account ID
+    /// @param previousOwner The previous account owner
+    /// @param newOwner The new account owner
+    event AccountOwnershipTransferred(
```

Comment:
```suggestion
    event AccountTransferred(
```
I would prefer shorter names in general.  I think, we can just call it like that.

Wdyt?

Reply:
I agree, they're too verbose, fixed, `transfer account ownership` is now `transfer account`.

Reply:
cool